receiver/awscontainerinsightreceiver/config.go

```diff
@@ -43,4 +43,9 @@ type Config struct {
 	// ClusterName can be used to explicitly provide the Cluster's Name for scenarios where it's not
 	// possible to auto-detect it using EC2 tags.
 	ClusterName string `mapstructure:"cluster_name"`
+
+	// The "LeaderLockName" is an optional attribute to override the name of the locking resource (e.g. config map) used during the leader
+	// election process for EKS Container Insights. The elected leader is responsible for scraping cluster level metrics.
+	// The default value is "otel-container-insight-clusterleader".
+	LeaderLockName string `mapstructure:"leader_lock_name"`
```

Comment:
sanity check: There's no situation where this lock name will be ignored/irrelevant right? i.e it is independent from the other attributes in the object

Reply:
Thats right. It is independent from the other config options.